src/main/java/io/snyk/jenkins/trasnform/ReportConverter.java

```diff
@@ -0,0 +1,34 @@
+package io.snyk.jenkins.trasnform;
+
+import javax.annotation.Nonnull;
+import java.util.Objects;
+
+import jodd.jerry.Jerry;
+
+public class ReportConverter {
+
+  private final Jerry.JerryParser parser;
+
+  private static class Helper {
+    private static final ReportConverter INSTANCE = new ReportConverter();
+  }
+
+  private ReportConverter() {
+    parser = Objects.requireNonNull(Jerry.jerry());
+  }
+
+  @Nonnull
+  public static ReportConverter getInstance() {
+    return Helper.INSTANCE;
+  }
+
+  public String modifyHeadSection(String htmlWithInlineCSS) {
+    Jerry document = parser.parse(htmlWithInlineCSS);
+
+    // remove inline css (it's disabled by jenkins)
+    document.$("head").$("style").remove();
```

Comment:
Can we make this optional, so that if the report has no style element, we’re still ok?

This way we can make changes to snyk-to-html without breaking this plugin.

Reply:
@adrukh, we can do it optional if you want, but if report has no style element, nothing will be removed (no exceptions too). See the unit test https://github.com/jenkinsci/snyk-security-scanner-plugin/pull/25/files#diff-98c57eeb055b8721c6b5612018c97e2cR13

Reply:
Very well!